packages/client/src/clients/kamiden/proto.ts

```
@@ -128,7 +129,8 @@ export interface RoomRequest {
   Size?: number | undefined;
 }
 
-export interface StreamRequest {}
+export interface StreamRequest {
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)